src/Router.tsx

```diff
@@ -14,8 +16,12 @@ const router = createBrowserRouter([
         path: "test",
         element: <Home />,
       },
+      {
+        path: "posts",
```

Comment:
사소하지만 개인적으로는 요렇게 써주는게 더 명확한 것 같아서 좋은 것 같아요!
```suggestion
        path: "/posts",
```